src/style-spec/reference/v8.json

```diff
@@ -6061,6 +6070,82 @@
       },
       "property-type": "data-constant"
     },
+    "raster-color": {
+      "type": "color",
```

Comment:
@rreusser should we provide a default value similarly to heatmap-color
```
      "default": [
        "interpolate",
        [
          "linear"
        ],
        [
          "heatmap-density"
        ],
        0,
        "rgba(0, 0, 255, 0)",
        0.1,
        "royalblue",
        0.3,
        "cyan",
        0.5,
        "lime",
        0.7,
        "yellow",
        1,
        "red"
      ],
```

Reply:
I don't have strong feelings on the default. I thought maybe grayscale was reasonable because we can really just say a lot less about the meaning of the data than we can for heatmap (which is necessarily point density).